examples/java/src/main/java/glide/examples/ClusterExample.java

```diff
@@ -0,0 +1,165 @@
+        String host = "localhost";
+        Integer port = 6379;
+        // GLIDE is able to detect all cluster nodes and connect to them automatically
+        // even if only one of them was configured
+
+        // Check `GlideClusterClientConfiguration` for additional options.
+        GlideClusterClientConfiguration config =
+                GlideClusterClientConfiguration.builder()
+                        .address(
+                                NodeAddress.builder()
+                                        .host(host)
+                                        .port(port)
+                                        .build())
+                        // Enable this field if the servers are configured with TLS.
+                        //.useTLS(true);
+                        .build();
+
+        GlideClusterClient client = GlideClusterClient.createClient(config).get();
+        return client;
+    }
+
+    /**
+     * Executes the main logic of the application, performing basic operations such as SET, GET, PING,
+     * and INFO REPLICATION using the provided <code>GlideClusterClient</code>.
+     *
+     * @param client An instance of <code>GlideClusterClient</code>.
+     * @throws ExecutionException if an execution error occurs during operations.
+     * @throws InterruptedException if the operation is interrupted.
+     */
+    public static void appLogic(GlideClusterClient client) throws ExecutionException, InterruptedException {
+
+        // Send SET and GET
+        CompletableFuture<String> setResponse = client.set("foo", "bar");
+        Logger.log(Logger.Level.INFO, "app", "Set response is " + setResponse.get());
+
+        CompletableFuture<String> getResponse = client.get("foo");
+        Logger.log(Logger.Level.INFO, "app", "Get response is " + getResponse.get());
+
+        // Send PING to all primaries (according to Valkey's PING request_policy)
+        CompletableFuture<String> pong = client.ping();
+        Logger.log(Logger.Level.INFO, "app", "Ping response is " + pong.get());
+
+        // Send INFO REPLICATION with routing option to all nodes
+        ClusterValue<String> infoResponse =
+                client.info(
+                        InfoOptions.builder().section(InfoOptions.Section.REPLICATION).build(), ALL_NODES).get();
+        Logger.log(
+                Logger.Level.INFO,
+                "app",
+                "INFO REPLICATION responses from all nodes are " + infoResponse.getMultiValue());
+    }
+
+    /**
+     * Executes the application logic with exception handling.
+     *
+     * @throws ExecutionException if an execution error occurs during operations.
+     */
+    private static void execAppLogic() throws ExecutionException {
+
+        while (true) {
+            GlideClusterClient client = null;
+            try {
+                client = createClient();
+                appLogic(client);
```

Comment:
return appLogic(client);

Reply:
Cannot return a value from a method with void result type. Can I propose the following?
```suggestion
                appLogic(client);
                return;
```